packages/connect/src/router.ts

```diff
@@ -52,17 +53,31 @@ import type { ProtocolHandlerFactory } from "./protocol/protocol-handler-factory
  */
 export interface ConnectRouter {
   readonly handlers: UniversalHandler[];
+  /**
+   * Provides implementation for a set of RPCs on the service.
+   */
   service<T extends ServiceType>(
     service: T,
     implementation: Partial<ServiceImpl<T>>,
     options?: Partial<UniversalHandlerOptions>,
   ): this;
+  /**
+   * Provides implementation for a single RPC given service and associated method.
+   */
   rpc<M extends MethodInfo>(
     service: ServiceType,
     method: M,
     impl: MethodImpl<M>,
     options?: Partial<UniversalHandlerOptions>,
   ): this;
+  /**
+   * Provides implementation for a single RPC given a method type.
```

Comment:
I'm in favor of the proposed connect-query API, but we're still working with a draft PR with "experimental" in the name. This new feature will be out of place if we don't ship the new API for whatever reason.

Can we mark it `@private`? I wouldn't have any concerns publishing it then.

```suggestion
   * Provides implementation for a single RPC given a method type.
   * 
   * @private This is an experimental API. Please do not rely on it yet.
```